File: 0058_length_of_last_word/Solution.java
class Solution {
    public int lengthOfLastWord(String s) {
        String str = s.trim();
        int l = 0;

        for (int i = str.length() - 1; i >= 0; i--) {
            if (str.charAt(i) == ' ') {
                break;
            }

            l++;
        }

        return l;
    }
}

File: 0058_length_of_last_word/Solution.js
var lengthOfLastWord = function (s) {
    let str = s.trim();
    let l = 0;

    for (let i = str.length - 1; i >= 0; i--) {
        if (str.charAt(i) == ' ') {
            break;
        }

        l++;
    }

    return l;
};
